duckduckgo/duckduckgo-privacy-extension
# Move background and content script files around

**Reviewer:** @jdorweiler 



## Description:

Move files around so they make more sense (specifically separating content scripts from background files and class-based background files from singleton ones).

I've also updated the file naming scheme to match that of the UI code.

The folder structure is now:

```
shared/js/
|- ui/
|- content-scripts/
|- background/
|--- classes/
```

## Steps to test this PR:

1. You may need to do `rm -r build/` (otherwise the Makefile might get confused)
2. The extension should build correctly (any incorrect linkage should immediately come up when running e.g. `npm run dev-chrome`)
3. The two content scripts should be added without any errors:
  - oninstall: when the extension is installed and `https://duckduckgo.com/app?post=1` is loaded
  - socialwidgets: on any page

## Automated tests:

###### Reviewer Checklist:


###### PR Author Checklist:
